types/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import { ArcTables } from "./tables";
+
+export const tables: ArcTables;
```

Comment:
This isn't quite right, it ends up as the concrete type `ArcTables<AnyTables>`. We want this to be a generic function but I'm not sure how to export it as such.